glsk/glsk-document-api/src/main/java/com/powsybl/glsk/api/GlskShiftKey.java

```diff
@@ -70,4 +72,9 @@ public interface GlskShiftKey {
     String getFlowDirection();
 
     double getMaximumShift();
+
+    /**
+     *  setter time interval of shift key
+     */
+    void setGlskShiftKeyInterval(Interval interval);
```

Comment:
Why do you add that ? It feels like you don't use it afterwards ? (no getter)

Reply:
After investigation it seems that actually nobody is using this property, so you could even remove it from AbstractGlskShiftKey. At least, I think you can remove the setters and remove the part where you set this value in your code.